### PR TITLE
Update left-bar.component.html

### PR DESCRIPTION
**Pull request by CoderDrShen**

### CODE REVIEW

**Review by lazynina — APPROVED** [review]

**Comment by maebeam** [comment]
image
thoughts?

**Comment by lazynina** [comment]
> image
> thoughts?

I like $CLOUT!

File: src/app/left-bar/left-bar.component.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
       <left-bar-button
         [link]="'/' + globalVars.RouteNames.BUY_BITCLOUT"
-        [buttonLabel]="'Buy BitClout'"
+        [buttonLabel]="'Buy Clout'"
```
Comment (lazynina) [inline src/app/left-bar/left-bar.component.html:48]:
```suggestion
        [buttonLabel]="'Buy $CLOUT'"
```

File: src/app/left-bar/left-bar.component.html [hunk 2]
```diff
@@ -57,7 +57,7 @@
 
       <left-bar-button
         [link]="'/' + globalVars.RouteNames.SEND_BITCLOUT"
-        [buttonLabel]="'Send BitClout'"
+        [buttonLabel]="'Send Clout'"
```
Comment (lazynina) [inline src/app/left-bar/left-bar.component.html:60]:
```suggestion
        [buttonLabel]="'Send $CLOUT'"
```

**Review by lazynina — CHANGES_REQUESTED** [review]

File: src/app/left-bar/left-bar.component.html (hunk 2 above)
Comment (CoderDrShen) [reply src/app/left-bar/left-bar.component.html:60]: 😍😍

Comment (lazynina) [reply src/app/left-bar/left-bar.component.html:60]: If you like these changes, accept them and this PR should be good to go.

**Review by maebeam — APPROVED** [review]

**Review by lazynina — APPROVED** [review]

**Pull request merged**